tests/IntegrationTests/RevIntegrationTest.php

```diff
@@ -0,0 +1,240 @@
+<?php
+
+namespace RevAPI;
+
+class RevIntegrationTest extends \PHPUnit_Framework_TestCase
+{
+    const MEDIA_URL = 'http://mediahub.unl.edu/uploads/a07d73f214fe6bacbd446e6b90be8aa9.mp4';
```

Comment:
It might be a good idea to avoid a constant here and instead add this to test configuration. That will make it more future-proof.